modules/core/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsManager.java

```diff
@@ -24,9 +27,17 @@
     /**
      * Gather object statistics.
      *
-     * @param targets Target to gather statistics by.
+     * @param targets Target to params map to gather statistics by.
      * @throws IgniteCheckedException  Throws in case of errors.
      */
+    public void collectStatistics(StatisticsObjectConfiguration... targets) throws IgniteCheckedException;
```

Comment:
I guess we have to drop old method collectStatistics(StatisticsTarget..).
Please run and fix test